Gruntfile.js

```diff
@@ -96,8 +96,9 @@ module.exports = function(grunt) {
       },
       secondary: {
         files: {
-          'build/ddocs/medic/_attachments/ddocs/compiled.json':
-            'build/ddocs/medic-*/',
+          'build/ddocs/medic/_attachments/ddocs/medic.json': 'build/ddocs/medic-db/*',
+          'build/ddocs/medic/_attachments/ddocs/sentinel.json': 'build/ddocs/sentinel-db/*',
```

Comment:
Nit: we should update the `secondary-ddocs` watch regex so it captures changes to the new ddocs as well :)

Reply:
Absolutely! Great spotting.